test/Microsoft.ML.PerformanceTests/FeaturizeTextBench.cs

```diff
@@ -25,9 +25,8 @@ public class FeaturizeTextBench : BenchmarkBase
         [GlobalSetup]
         public void SetupData()
         {
-            Path.GetTempFileName();
```

Comment:
Do any of these tests need to change? I think it would be fine doing what they do today.

Reply:
Sounds good. I will go ahead and revert them then.

Reply:
Tests have been reverted.